File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1099.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');
    
n = parseInt(lines.shift());
for(i=0;i<n;i++){
    s = lines.shift()
    ns = s.split(' ')
    x = parseInt(ns[0])
    y = parseInt(ns[1])
    si=0;
    if(x>y){
        temp=x;
        x=y;
        y=temp;
    }
    if(x%2==0){
        x++;
    } else{
        x+=2;
    }
    for(j=x;j<y;j+=2){
        si+=j;
    }
    console.log(si)
}

File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1099.py
n = int(input())
i=0
while i<n:
    ns = input().split(' ')
    x = int(ns[0])
    y = int(ns[1])
    si=0
    if(x>y):
        temp=x
        x=y
        y=temp
    if(x%2==0):
        x=x+1
    else:
        x=x+2
    j=x
    while j<y:
        si=si+j
        j=j+2

    print(si)
    i=i+1
